packages/beta/src/__tests__/api/filesMultiPart.unit.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { sleepPromise } from '@cognite/sdk-core';
-import { mockBaseUrl } from '@cognite/sdk-core/src/__tests__/testUtils';
```

Comment:
only by changing import have been able to make the test work. 
so need to check for extra export config